### ISSUE TITLE
Grades

### ISSUE THREAD

**Issue by slugguts** [bethel-school-of-technology/bethel-bakery-backend#10]
Oh well. Could be worse.

**Comment by slugguts** [comment]
https://github.com/bethel-school-of-technology/bethel-bakery-backend/issues
https://github.com/bethel-school-of-technology/bethel-bakery-frontend/issues
Especially:
https://github.com/bethel-school-of-technology/bethel-bakery-frontend/issues/9

**Comment by slugguts** [comment]
I understand that it can be difficult to work with somebody like me, but not responding or engaging in a single communication/question makes me doubt that there is enough of a communication or accurate information for anyone to give a reasonable assessment of a grade.

While Populi introductions give a big emphasis on communication, I'm not seeing that acted out when it comes to me.

So is it just me? Nobody wants to talk to me? and I'm a bad person who deserves bad grades?

**Comment by slugguts** [comment]
It's apparent to me nobody cares about team Zion. So if I've blown the other 5 weeks by not coding, so be it. I tried to do what I thought was the requirement, and I failed. So says my grade. I know I can make an ecommerce website like I can bake a Betty Crocker cake so I'll have a project ready to demo by the end of the week, team Zion or no. Grade, or no.

I'm going to have a fun descriptive example for anyone who attends the demo, complete with the mistakes I made, commented out, with descriptions, and the solutions, and anecdotes to go along...it will be great! and enjoyable for all. I guess this is an individual project and grade, for me.

Talking to myself...crazy? Yes.
Setting up a reference for future generations, to show them the way, make it easier on them, and help them understand the chief difficulties in coding?

That is what coding is all about!
This is my github library for a group project that has been living in a mobile home on tornado alley during monsoon season.
I will update this post with the demo date!...for myself...and posterity.
